# Site message
string site_name
Measurement[] measurements
# geometry_msgs/Polygon area maybe later
uint8 has_pano